sdk/communication/AzureCommunicationCalling/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 2.0.1-beta.1 (2021-09-09)
+
+### Bugs Fixed
+- When trying to accept a terminated call `accept` API call will fail with appropriate error.
+- SDK rebuilt with latest available AzureCommunicationCommon `1.0.2` ([#344](https://github.com/Azure/Communication/issues/344)).
+- Using `CallKit` implementation in SDK , accepting another call when already in an active call failed.
```

Comment:
This technical detail isn't relevant to users. Would recommend folding into the above.

Reply:
They are 2 completely different bugs

Reply:
I will try to change the text